src/components/key/Key.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prefer-stateless-function, no-useless-constructor */
+import './Key.scss';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Key extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { op } = this.props;
+    const { display } = this.props;
+    return (
+      <div className={op ? 'calKey opColor' : 'calKey'}>
+        {display}
+      </div>
+    );
```

Comment:
- [ ] Clicking any of the buttons triggers this error with no output to the result screen.
![image](https://user-images.githubusercontent.com/33767951/148281172-e44cbad2-ed4a-4b27-bcb8-d619dd6bcb8c.png)

Reply:
Thanks @omar-labana for your review,

I can't locate the bug on my version! :thinking: 
Could you help me reproduce the error on my end ? 
like give me steps to show the error message.

Reply:
Thanks @omar-labana for your Slack feedback
i was able to reproduce the error after cloning the repo, install depends, start server.